op-guide/requirement.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Software and Hardware Requirements
+category: deployment
+---
+
+# TiDB Software and Hardware Requirements
+
+## About
+
+As an open source distributed NewSQL database with high performance, TiDB can be deployed in the Intel architecture server and mainstream virtualization environments and runs well. TiDB supports most mainstream hardware networks and the mainstream Linux operating system. 
+
+## Linux OS Version Requirements
+
+| Linux OS Platform        | Version      |
+| :-----------------------:| :----------: |
+| Red Hat Enterprise Linux | 7.3 and above|
+| CentOS                   | 7.3 and above|
+| Oracle Enterprise Linux  | 7.3 and above|
+| Ubuntu LTS               | 14.04 and above|
+
+**Note**
+
+- The Oracle Enterprise Linux only supports the Red Hat Compatible Kernel (RHCK), and does not support the Unbreakable Enterprise Kernel provided by Oracle Enterprise Linux.
+- The support for the Linux operating systems above include the deployment and operation in physical servers as well as in the mainstream virtualized environments like VMWare, KVM and XEM.
+
+## Server Requirements
+
+You can deploy and operate TiDB on the 64-bit universal hardware server platform in the Intel x86-64 architecture. The requirements and recommendations about server hardware configuration for development, testing and production environments are as follows:
+
+### Development and Testing Environments
+
+| Component | CPU     | Memory | Local Storage  | Network  | Instance Number (lowest requirement) |
+| :------: | :-----: | :-----: | :----------: | :------: | :----------------: |
+| TiDB    | 16 core+   | 16 GB+  | SAS, 200 GB+ | Gigabit network card | 1       |
+| PD      | 16 core+   | 16 GB+  | SAS, 200 GB+ | Gigabit network card | -       |
+| TiKV    | 16 core+   | 32 GB+  | SAS, 200 GB+ | Gigabit network card | 3       |
+|         |         |         |              | Total Server Number |  4      |     
+
+**Note**
+
+- Verify that in the test environment the TiDB and PD can be deployed on the same server.
```

Comment:
Verify that in the test environment the TiDB and PD can be deployed on the same server.
-> 
In the test environment, the TiDB and PD can be deployed on the same server.